model/task_history.go

```diff
@@ -233,3 +235,17 @@ func getOlderActiveMainlineTask(ctx context.Context, t task.Task) (*task.Task, e
 	// olderActiveTask can be nil, since it may not exist.
 	return olderActiveTask, nil
 }
+
+func GetTaskDateOrder(ctx context.Context, date time.Time, opts FindTaskHistoryOptions) (int, error) {
```

Comment:
Can you please add a function comment since this function is exported?

Reply:
oh sorry yes, I will add comments to all the functions I moved